docs/content/guides/developer/app-examples/trustless-swap.mdx

```diff
@@ -996,49 +1068,151 @@ The UI design consists of three parts:
 - A place for users to manage their owned objects to be ready for escrow trading called `Manage Objects`.
 - A place for users to discover, create, and execute trades called `Escrows`.
 
-:::warning
+### Scaffold a new app
 
-The following code snippets are not the full source code. The snippets are meant to focus on relevant logic important to the functionality of the example and features of Sui.
+The first step is to set up the client app. Run the following command to scaffold a new app.
 
-:::
+```bash
+pnpm create @mysten/dapp --template react-client-dapp
+```
 
-### Set up providers {#set-up-providers}
+or
 
-Set up and configure several providers at the root of your React.js tree to ensure different libraries including `dApp Kit`, `@tanstack/react-query`, `react-router-dom` work as expected.
+```bash
+yarn create @mysten/dapp --template react-client-dapp
+```
 
-```ts title='src/main.tsx'
-import { createNetworkConfig, SuiClientProvider, WalletProvider } from '@mysten/dapp-kit';
-import { getFullnodeUrl } from '@mysten/sui/client';
-import { Theme } from '@radix-ui/themes';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
-import { RouterProvider } from 'react-router-dom';
+### Setting up import aliases
 
-import { router } from '@/routes/index.tsx';
+First, set up import aliases to make the code more readable and maintainable. This allows you to import files using `@/` instead of relative paths.
 
-const queryClient = new QueryClient();
+Replace the content of `tsconfig.json` with the following:
 
-const { networkConfig } = createNetworkConfig({
-	localnet: { url: getFullnodeUrl('localnet') },
-	devnet: { url: getFullnodeUrl('devnet') },
-	testnet: { url: getFullnodeUrl('testnet') },
-	mainnet: { url: getFullnodeUrl('mainnet') },
-});
+{@inject: examples/trading/frontend/tsconfig.json}
```

Comment:
This is outputting the correct classes for json highlighting seemingly. I must need to import a style somewhere. Will look into that.